gometalinter.json

```diff
@@ -1,6 +1,6 @@
 {
   "Vendor": true,
-  "Deadline": "2m",
+  "Deadline": "5m",
```

Comment:
Was there a reason for changing this one?

Reply:
Change it just because of the lint job.

*  [before](https://circleci.com/gh/docker/cli/24297#build-timing/containers/0)
    
The log content:

 ```
cli/command/formatter/reflect_test.go:15:34:warning: nolint directive did not match any issue (nolint)
WARNING: deadline exceeded by linter unused (try increasing --deadline)
Exited with code 3
```
* [after](https://circleci.com/gh/docker/cli/24302#build-timing/containers/0)

The Job been successful.

Change it to `5m` , as I said above. https://github.com/docker/cli/pull/1818#issuecomment-481953662

```
/go/src/github.com/docker/cli # time /usr/local/bin/gometalinter --config=gometalinter.json ./...
real    4m 55.06s
user    1m 55.14s
sys     0m 26.43s
```
Of course, maybe the machine I tested is of poor performance. :thinking:  I can drop this change.

Reply:
Let's drop it for now; if it's become slower because of this update, that might be important to know

Reply:
sure, I will update it now.

Reply:
Addressed. Thanks. :+1: